wire-ios-data-model/Source/Model/User/ZMUser+LegalHoldRequest.swift

```diff
@@ -272,4 +276,42 @@ extension ZMUser: SelfLegalHoldSubject {
         needsToAcknowledgeLegalHoldStatus = false
     }
 
+    // MARK: - Fingerprint
+
+    var proteusProvider: ProteusProviding? {
+        if let context = self.managedObjectContext?.zm_sync {
+            return ProteusProvider(context: context)
+        }
+        return nil
+    }
```

Comment:
maybe move this to managedObjectContext extension as we have the exact code on UserClient